oct/ansible/oct/roles/docker/tasks/configure_docker_daemon_storage.yml

```diff
@@ -1,40 +1,18 @@
 ---
-- name: create a logical volume for Docker data
-  lvol:
-    vg: '{{ origin_ci_docker_volume_group }}'
-    lv: 'docker-data'
-    size: '70%FREE'
```

Comment:
The VG size is configurable but is usually 20GB I believe. I think due to the weirdness with the openshift XFS vol dir using the remaining data we had configured this split -- I would like maybe @danmcp or @bparees to pitch in here with why we chose the percentage splits the way we did and if it would be OK to use the defaults, not sure I know the original justification for those changes.